.travis.yml

```diff
@@ -4,17 +4,19 @@ mono: none
 dotnet: 2.2
 dist: xenial
 cache:
+  yarn: true 
   directories:
     - $HOME/.nuget/packages
+    - $HOME/.chakracore
 before_install:
   - bash scripts/install_protobuf.sh
   - nvm install node
   - npm install -g yarn
 install:
   - export DOTNET_CLI_TELEMETRY_OPTOUT=1
 before_script:
-  - bash scripts/download_chakracore.sh 1_11_1
-  - export LD_LIBRARY_PATH=$(pwd)/ChakraCoreFiles/lib/:$LD_LIBRARY_PATH
+  - export LD_LIBRARY_PATH=$HOME/.chakracore/1_11_1/:$LD_LIBRARY_PATH
```

Comment:
https://docs.travis-ci.com/user/environment-variables/

use environment, to avoid write version twice